sqls/db/db-objects.sql

```diff
@@ -140,6 +140,7 @@ CREATE TABLE [dbo].[Subject](
     [photoUrl] [varchar](512) NULL,
     [notes] [varchar](max) NULL,
 	[status] [tinyint] NULL,
+	[hash] [char](40) NULL,
```

Comment:
wouldn't be better to be VARCHAR?

Reply:
It has a fixed length, why it should be VARCHAR then?

Reply:
I was thinking that if we change the way we compute the hash in the future we will not have to update the DB also. And the Hash.digest('hex') output length isn't 32?

Reply:
It is 40, I've checked.